README.md

```diff
@@ -2,22 +2,23 @@
 
 # Project Name
```

Comment:
[Optional] You should have mentioned that the placeholder `Project Name` be changed with the actual name of the project in the ReadMe.